React-Template-master/src/theme/palette.ts

```diff
@@ -66,10 +66,8 @@ const ERROR: PaletteColorOptions = {
     contrastText: '#fff'
 };
 
-export const BUTTONCOLORS: SimplePaletteColorOptions = {
-    light: '#EBEBF0',
-    dark: '#3E0EC3',
-    main: ''
+export const WHITE = {
```

Comment:
There is no need to define white color as it is already present in the `common`. See line 87

Reply:
will fix this